### PR TITLE
Ps 8.0 5139 5031

### PR DESCRIPTION
**Pull request by george-lorch**

### CODE REVIEW

**Comment by george-lorch** [comment]
https://ps.cd.percona.com/view/8.0/job/percona-server-8.0-param/93/

File: mysql-test/suite/rocksdb/t/add_index_inplace.test [hunk 1]
```diff
@@ -398,4 +398,33 @@ ALTER TABLE t1 ADD INDEX kb(b), ALGORITHM=INPLACE;
...
+
+ALTER TABLE t1 ALTER INDEX kb INVISIBLE;
+SHOW CREATE TABLE t1;
+
+ALTER TABLE t1 ALTER INDEX kb VISIBLE;
+SHOW CREATE TABLE t1;
+
+--let $end_max_index_id = query_get_value(SELECT * from INFORMATION_SCHEMA.ROCKSDB_GLOBAL_INFO where type = 'MAX_INDEX_ID', VALUE, 1)
+
+--let $assert_text= new new index ids allocated
```
Comment (laurynas-biveinis) [inline mysql-test/suite/rocksdb/t/add_index_inplace.test]: "no new?"

**Review by laurynas-biveinis — APPROVED** [review]
just one typo

File: mysql-test/suite/rocksdb/t/add_index_inplace.test (hunk 1 above)
Comment (george-lorch) [reply mysql-test/suite/rocksdb/t/add_index_inplace.test]: Ack, will fix the text before merge.

**Pull request merged**